Makefile

```diff
@@ -4,7 +4,7 @@ ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
 PRODUCT := xchammer.app
 # This is fragile; even on arm64, it defaults to x86_64, and the path could easily change.
-XCHAMMER_APP := $(ROOT_DIR)/bazel-out/applebin_macos-darwin_x86_64-dbg-ST-639799358421/bin/xchammer.app
+XCHAMMER_APP := $(ROOT_DIR)/bazel-out/applebin_macos-darwin_x86_64-opt-ST-639799358421/bin/xchammer_archive-root/xchammer.app
```

Comment:
We'll need to split this apart into separate variables then; I was using it with `make dev` which requires the debug build.

Reply:
Also doesn't this break if you switch from debug to release BTW?

Reply:
I don't know if it was always like this. I just can't test `make install` to then try to invoke `xchammer install_xcode_build_system` because the install cmd gives me this err:
```sh
mkdir -p /usr/local/bin
ditto /Users/thiago/Development/xchammer/bazel-out/applebin_macos-darwin_x86_64-dbg-ST-639799358421/bin/xchammer.app /usr/local/bin/xchammer.app
ditto: Cannot get the real path for source '/Users/thiago/Development/xchammer/bazel-out/applebin_macos-darwin_x86_64-dbg-ST-639799358421/bin/xchammer.app'
make: *** [install] Error 1
```

So was trying to find a way to make that work

Reply:
Yeah that makes sense to me, I was more of suggesting that we shouldn't hardcode the fragment if it's not necessary. Concretely, if you switch to a release build in some way you'll hit this issue.